sdk/attestation/azure-security-attestation/src/private/crypto/CMakeLists.txt

```diff
@@ -2,7 +2,6 @@
 # SPDX-License-Identifier: MIT
 
 cmake_minimum_required (VERSION 3.13)
-project(azure-security-attestation-crypto LANGUAGES CXX)
```

Comment:
Removing this line hides the folder from the project tree and makes it visible within the attestation folder